index.js

```diff
@@ -39,6 +39,7 @@ module.exports = {
     esnext: require('./lib/config/esnext'),
     react: require('./lib/config/react'),
     typescript: require('./lib/config/typescript'),
+    'typescript-fast': require('./lib/config/typescript-fast'),
```

Comment:
The full-fat version will likely have a significant performance impact on web.

Reply:
Cool :) In that case if we keep it, what do you think to making it be a base-level config - so you import typescript or typescript-fast instead of needing both?